src/renderer/components/widgets/channels/ChannelInput/ChannelInput.test.tsx

```diff
@@ -43,9 +37,8 @@ describe('ChannelInput', () => {
                     <div
                       class="makeStyles-input-3"
                       contenteditable="true"
-                      placeholder="Message test"
+                      placeholder="Message Message #channel as @user"
```

Comment:
typo, i think?

Reply:
Not exactly. Apparently component adds "Message" string at the beginning of the placeholder automatically. Then, test renders DOM which contains doubled "Message" as it's also part of the input prop. In fact it proves that component works properly and that "typo" is only cosmetic. Real components has right placeholders set.

Reply:
This seems like it's asking for trouble/confusion but ok?